### PR TITLE
Adds skip link to on-demand header

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
This PR adds the necessary CSS and JS to support skip links. Since this
header is going on different pages with unique page layouts we can't
hook into #main the same way we do on v1 pages. Instead there's a blank
div at the end of the navigation that will allow skip-nav users to skip
the navigation.

## Additions

- Add skip link to on demand header

## Testing
- run `gulp scripts:ondemand && gulp styles:ondemand`
- `http://localhost:8000/test-fixture/?atomic=header` and tab to activate the skip-nav button

## Review

- @anselmbradford 
- @jimmynotjim 
- @sebworks 

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by sebworks** [comment]
👍

File: cfgov/unprocessed/css/on-demand/header.less [hunk 1]
```diff
@@ -43,6 +43,10 @@
    ========================================================================== */
 @import (less) "../misc.less";
 
+/* Template pieces
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/on-demand/header.less]: Minor: Can we move this edit below the `Organism pieces` section so it follows the cascade of atom > molecule > organism > template, and also follows the format in main.less.

**Comment by jimmynotjim** [comment]
👍

File: cfgov/unprocessed/css/on-demand/header.less (hunk 1 above)
Comment (KimberlyMunoz) [inline cfgov/unprocessed/css/on-demand/header.less]: Sure.

**Pull request merged**